e2e/browser/src/components/appContainer/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import {
+  login,
+  logout,
+  handleIncomingRedirect,
+  getDefaultSession,
+} from "@inrupt/solid-client-authn-browser";
+import Notifications from "../notifications/";
+
+const REDIRECT_URL = "http://localhost:3000/";
+const APP_NAME = "Notifications browser-based tests app";
+const issuer = "https://login.inrupt.com/";
```

Comment:
ideally this should come from an environment variable, along with docs as to why it's this specific environment

Reply:
There are some changes to this intent in #232, I'll add this there.

Reply:
Actually the issuer does come from the environment (it is set by playwright), the app name is not environment-specific, and once using `window.location.href` neither is the redirect URL.

Reply:
Yup, you just need to call this `DEFAULT_ISSUER` instead of `issuer` and then you can use it as the initial value in your `useState` for the issuer

Reply:
Done in https://github.com/inrupt/solid-client-notifications-js/pull/229/commits/aa86165ebd61620ee42d0c6cdf9a27538606b1fe :)